### ISSUE TITLE
copy comment's link

### ISSUE THREAD

**Issue by yashpalgoyal1304** [TeamNewPipe/NewPipe#7378]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
🔍 [copy comment link](https://github.com/teamnewpipe/newpipe/issues?q=is%3Aissue+copy+comment+link)
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?

Ability to copy a comment's link

#### Why do you want this feature?

Many comments contain very useful info. and since u cant interact with comments in np app, so, this feature becomes even more crucial to open the concerned link directly elsewhere.

**Comment by vishalnandagopal** [comment]
I don't think there is a way to link to a specific comment on YouTube. YouTube doesn't assign links to each comment like Reddit, Twitter or GitHub.

**Comment by yashpalgoyal1304** [comment]
> There's no way to link to a specific comment on YouTube. YouTube doesn't assign links to each comment

oh lol, no. youtube assigns specific comment on youtube from as far ago as i could remember

**Comment by vishalnandagopal** [comment]
> youtube assigns specific links to comments 

 Can you provide an example link? I've never seen one so not sure of their format or what they look like.

> try clicking a comment from youtube notifications (on desktop at least)

On desktop, if I click on a notification mentioning a reply, it opens a mini popup of the comments of the video in the same notification popup.

> notice that particular comment shows up  and has the label "highlighted comment" over it.

That's just the web app showing your comment on the top. The label is to help distinguish the comment from other comments.

**Comment by yashpalgoyal1304** [comment]
![image](https://user-images.githubusercontent.com/19423063/141673359-26933e25-e3f2-4c4b-920c-8c7e9c2ee317.png)

----------
oh lol, anyways:

example link: 
https://www.youtube.com/watch?v=ussCHoQttyQ&lc=UgzyJkv0nrIaeq9bUGB4AaABAg.8bXXdqtNDAF8cF-C7SN4yz

```
v=ussCHoQttyQ
lc=UgzyJkv0nrIaeq9bUGB4AaABAg
.8bXXdqtNDAF8cF-C7SN4yz
```

![image](https://user-images.githubusercontent.com/19423063/141673395-d94e306a-8f1c-4edb-a2f2-8440f51c1117.png)

**Comment by vishalnandagopal** [comment]
Yeah these links worked, thanks for correcting me.

**Comment by yashpalgoyal1304** [comment]
yeah, i have incurred many ppl before who arent aware about youtube comment links. and it feels yet another soooooo basic thing to me, and my mind is smtimes like how can ppl miss this/that. but uh i guess i am just.... whatever. anyways, glad that it worked and i could help (:

**Comment by yashpalgoyal1304** [comment]
but yeah, youtube also goes above and beyond to crumple the comment links, these yt comments links are effectively removed by youtube, they dont work on youtube app either, not sure if they work in yt web-phone

**Comment by hazysu** [comment]
Perhaps when you hold-tap on a comment you could choose to copy the link to the comment *or* the text? Similarly to how hold-tapping on a video in a list shows options in a context menu

**Comment by yashpalgoyal1304** [comment]
hi @vishalnandagopal would u mind hiding (`…` > `hide`) those comments? 'cz i dont think they contribute to the discussion. DONT delete though.
